packages/orchestrator-mcp/src/conflict-resolver.ts

```
@@ -0,0 +1,1036 @@
+          description: 'Clarified positions and underlying assumptions',
+          positions: positions.map(p => ({
+            ...p,
+            rationale: [...p.rationale, 'Clarified in mediation']
+          }))
+        };
+
+      case 'evaluate-evidence':
+        const evidenceScores = positions.map(p => ({
+          participant: p.participant,
+          score: this.scoreEvidence(p.evidence)
+        }));
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)